.eslintrc.yml

```diff
@@ -0,0 +1,92 @@
+extends:
+  - plugin:@typescript-eslint/eslint-recommended
+  - plugin:@typescript-eslint/recommended
+  - plugin:@typescript-eslint/recommended-requiring-type-checking
+  - plugin:import/errors
+  - plugin:import/warnings
+  - plugin:import/typescript
+  - plugin:promise/recommended
+  - airbnb-typescript
+  - plugin:jest/recommended
+  - plugin:jest/style
+  - plugin:prettier/recommended
+plugins:
+  - react-hooks
+  - unicorn
+parser: "@typescript-eslint/parser"
+parserOptions:
+  ecmaVersion: 6
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+    modules: true
+  project: "./tsconfig.json"
+env:
+  node: true
+  es6: true
+  browser: true
+rules:
+  # NOTE: any rule set to warning (warn or 1)
+  # eslint will NOT  output any warning when running `npm test` or `npm run eslint`
+  # warnings will still appear on the editor 
+  no-nested-ternary: 0
```

Comment:
I would personally prefer not to use nested ternaries. Do you find them useful?

Reply:
Yeah, I do find them useful, but I see the harm. I'm totally ok with disallowing.